src/app/src/Map.js

```diff
@@ -16,7 +16,14 @@ class GLMap extends Component {
                     {...this.props.viewport}
                     onViewportChange={this.props.updateViewport}
                     mapboxApiAccessToken={process.env.REACT_APP_MAPBOX_API_KEY}
-                />
+                    mapStyle={'mapbox://styles/alash/cjpconu8b0sas2qn5rx51uog8'}
```

Comment:
Would be good to consolidate this in an azavea or ism based account and remove it from alash specifically.  Probably fine to make an issue so it doesn't block this PR.

Reply:
Created #28 to track this.